### ISSUE TITLE
[Feature Request] Add "Native Gyro" command option to Steam Input

### ISSUE THREAD

**Issue by isaboll1** [ValveSoftware/steam-for-linux#8677]
Steam Input currently has options for allowing Gyro within controllers to emulate various forms of Input such as mouse input, as well as joystick axis inputs. From what I can tell, Steam Input relies on SDL2 in order to expose functionality as well.

SDL2 itself offers functions to capture gyro axis from respective controllers, and some games/emulators actually make use of these functions in order to enable gyro controller support. 

I am requesting for an option to be added in the Steam Input configurator for gyro input to be read as "Native Gyro", similar to the "Touchscreen Native" system command that's available for Steam Input on the Deck that allows for Touchscreen input to be recognized as actual multi touch, rather than mouse emulation. The "Native Gyro" command could expose the Steam Input controllers Gyro values to SDL2 as actual gyro axis, rather than having the Steam Virtual Controller exposed to SDL2 omit the gyro values only for mouse emulation or joystick axis emulation.

**Comment by isXander** [comment]
Vouch. This needs to be added and doesn't seem that difficult.

**Comment by enjarai** [comment]
This could definitely be useful for specific games/game mods that have native gyro support, it seems like an oversight to not have this option.

**Comment by cuesta4** [comment]
This would be great. Many emulators moved to SDL2 and stopped using cemuhook for gyro, so basically you cannot use Steam Input with RPCS3, Ryujinx, CEMU and others if you want gyro.

**Comment by jasonridesabike** [comment]
Please add! Let us gyrate!!

**Comment by jdfr228** [comment]
Also requesting this be added and subscribed to the issue. Seems a long time coming!

**Comment by joshbirnholz** [comment]
Also requesting this! This would be really great to have!

Cemuhook is _okay_ for using gyro in some emulators, but if you want to use any connected controllers other than the built-in controls, like when docked to a TV, you can't use the gyro unless you disable Steam Input entirely!

**Comment by nadanre89** [comment]
I would also really like to see this feature come to fruition. Being able to forward gyro of external controllers to anything ingame, or in-emulator would be so awesome.

**Comment by nadanre89** [comment]
How likely do you think it is, that Valve is going to implement it?

**Comment by tails101** [comment]
been 4 years, 2 different controllers (soon 3 maybe 4 controllers) and still nothing.
I probably won't buy another steam product and go for a 3rd party controller unless this bug is fixed.

**Comment by pepi55** [comment]
Lord gaben,

With your infinite wisdom and several yachts, please look once again to our mere mortals' pleas and grant us true gyro support for the new steam controller through steaminput!

I want to enjoy games which natively support gyro with the developer-implemented gyro support instead of having to hack and fiddle with joystick controls to mimic a fraction of its true power!

Thanks in advance 🙏

**Comment by tails101** [comment]
The other and easy solution would be to just let us use unblinded output with sdl3 so we can make hybrid solution.
So for example gyro is used by sdl but buttons by steam input.
So an option to unhide the controller used by steam input.

**Comment by isXander** [comment]
SDL3 now has full support for the Steam Controller, but it is frankly useless because Steam Input proxies the controller, rendering many games' advanced controller features like gyro, back button support, touchpad support, inoperable.

**Comment by nnnn20430** [comment]
@isXander Yep, trying to play a game that uses SDL3 with the new Steam Controller and it works great without Steam including gyro, but when added as a non Steam game and launched through Steam (the supposed recommended way to use the controller lol), gyro breaks, have to use Steam input "gyro to mouse", but it's awful, because I like to keep gyro always on in the game, but then the mouse input fights my stick inputs in menu's... also the mouse is invisible because mouse emulation is borked in Wayland for me.... so navigating menu's is a nightmare as it keeps selecting random options.
